CHANGELOG.md

```diff
@@ -20,13 +20,17 @@ The changelog format is based on [Keep a Changelog](https://keepachangelog.com/e
 
 ## [released]
 
-## [0.4.2] - 2023-02-05
-### Added
-- Deployment refactored to fit and follow new guidelines from DevSecOps team.
-- Data soverenity printing the contract id
-- 
+## [0.4.4] - 2023-02-21 
+###  Preparing for archiving the repository
```

Comment:
Will this repo actually be archived? Or is this a leftover from some fork?

Reply:
This is a left over from the other repo change log. We are preparing the v0.4.4 release change log still because of this we have it on draft